build.gradle

```diff
@@ -7,37 +7,44 @@ buildscript {
         }
     }
 
+    def sdkTargetJavaVersion = JavaVersion.VERSION_11
+    ext.versions = [
+            "javaSourceCompatibility": sdkTargetJavaVersion,
+            "javaTargetCompatibility": sdkTargetJavaVersion,
+    ]
+
```

Comment:
nit: looks like there may be an extra space here

Reply:
Fixed